lib/graphql/static_validation/rules/argument_literals_are_compatible.rb

```diff
@@ -41,7 +41,9 @@ def on_argument(node, parent)
               error_options = {
                 nodes: parent,
                 type: kind_of_node,
-                argument: node.name
+                argument: node.name,
+                argument_defn: arg_defn,
+                value: node.value
```

Comment:
I considered storing `GraphQL::Language::Printer.new.print(node.value)` instead but figured it is better to keep the higher fidelity value